### PR TITLE
Get domain from name of Record

### PR DESCRIPTION
**Pull request by ghost**
Get `domain` from `Record.name` if domain is not provided in request's data.
If `domain` is not found raise `ValidationError`.

### CODE REVIEW

**Comment by krzysztofzygmunt** [comment]
:ok:

File: dnsaas/api/v2/serializers.py [hunk 1]
```diff
@@ -119,6 +134,22 @@ def validate(self, attrs):
                     {'content': ['IP address cannot be private.']}
                 )
 
+        if not self.instance:
+            # get domain from name only for creation
+            if not domain and not attrs.get('name', None):
```
Comment (mkurek) [inline dnsaas/api/v2/serializers.py]: isn't name required argument?

File: dnsaas/api/v2/serializers.py [hunk 2]
```diff
@@ -61,6 +61,19 @@ def get_last_change(self, obj):
             return obj.last_change_json
 
 
+def hostname2domain(hostname):
```
Comment (mkurek) [inline dnsaas/api/v2/serializers.py:64]: add some docstring

Comment (ghost) [reply dnsaas/api/v2/serializers.py:64]: done

**Comment by mkurek** [comment]
:ok:

**Comment by szok** [comment]
:ok:

**Pull request merged**